cg/src/domains/subjects/subjects.routes.js

```diff
@@ -31,6 +35,12 @@ module.exports = app => {
     asyncHandler(processorsController.getProcessors.bind(processorsController))
   );
 
+  router.get(
+    '/data-shares/share',
```

Comment:
So a subject can:
- Create a URL to share his data
- Remove the URL
- Share the URL

Reply:
correct